lib/DiscoveryStreamFeed.jsm

```diff
@@ -865,6 +866,9 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
         Cu.reportError(`Error trying to load component feeds: ${error}`)
       ),
     ]);
+    if (!updateOpenTabs) {
+      dispatchQueue.forEach(e => this.store.dispatch(e));
```

Comment:
No need for the `if` as the queue is empty otherwise. Also consistent naming of `e` -> `action`

Reply:
Ah I see, yes, that makes sense.